hermes-consumers/src/test/groovy/pl/allegro/tech/hermes/consumers/consumer/result/undelivered/LogUndeliveredMessageHandlerSpec.groovy

```diff
@@ -0,0 +1,58 @@
+package pl.allegro.tech.hermes.consumers.consumer.result.undelivered
+
+import pl.allegro.tech.hermes.api.Subscription
+import pl.allegro.tech.hermes.common.config.ConfigFactory
+import pl.allegro.tech.hermes.common.config.Configs
+import pl.allegro.tech.hermes.common.exception.InternalProcessingException
+import pl.allegro.tech.hermes.common.kafka.KafkaTopicName
+import pl.allegro.tech.hermes.common.kafka.offset.PartitionOffset
+import pl.allegro.tech.hermes.common.message.undelivered.UndeliveredMessageLog
+import pl.allegro.tech.hermes.consumers.consumer.Message
+import pl.allegro.tech.hermes.consumers.consumer.sender.MessageSendingResult
+import spock.lang.Specification
+
+import java.nio.charset.StandardCharsets
+import java.time.Clock
+
+import static pl.allegro.tech.hermes.api.SentMessageTrace.createUndeliveredMessage
+import static pl.allegro.tech.hermes.consumers.consumer.sender.MessageSendingResult.failedResult
+import static pl.allegro.tech.hermes.consumers.test.MessageBuilder.withTestMessage
+
+
+class LogUndeliveredMessageHandlerSpec extends Specification {
+
+    private static final String MESSAGE_CONTENT = "ala ma kota"
```

Comment:
all of these are used once and definition block is longer than test itself.. how about inlining them?